lib/tests/snowflake/snowpark/dataframe.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+class DataFrame:
+    """This is dummy DataFrame class,
+    which imitates snowflake.snowpark.dataframe.Dataframe class
+    for testing purposes."""
+
+    __module__ = "snowflake.snowpark.dataframe"
+
+    def __init__(self):
+        pass
```

Comment:
To avoid confusion, let's add "mock" to these test class names: `MockSnowparkDataFrame` and `MockSnowparkRow`.

(It's not a big deal, but I'd also stick them together in the same source file, maybe called "SnowparkMocks" or something.)

Reply:
I've moved these classes to one file called `snowpark_mocks.py`, when I see it, it's better idea, thank you 👍 I didn't manage to change these names to `MockSnowparkDataFrame` and `MockSnowparkRow`, because then type checking with `is_type` didn't work, because `get_fqn_type` returns ex. `snowflake.snowpark.dataframe.MockSnowparkDataFrame` instead of `snowflake.snowpark.dataframe.DataFrame`